pkg/settings/settings.go

```diff
@@ -86,6 +86,7 @@ const (
 	SupportBundleExpirationSettingName                = "support-bundle-expiration"
 	NTPServersSettingName                             = "ntp-servers"
 	AutoRotateRKE2CertsSettingName                    = "auto-rotate-rke2-certs"
+	KubeconfigTTLSettingName                          = "kubeconfig-default-token-ttl-minutes"
```

Comment:
I think we need to add the setting, so we will have it by default.

```
KubeconfigTTL = NewSetting(KubeconfigTTLSettingName, "43200") // "43200" is default value in Rancher
```

Reply:
> I think we need to add the setting, so we will have it by default.
> 
> ```
> KubeconfigTTL = NewSetting(KubeconfigTTLSettingName, "43200") // "43200" is default value in Rancher
> ```

Exactly, i'm currently testing that.

Reply:
This will auto-create the `Settings` resource, but it is still not visible in the Harvester `Settings` UI.

```
$ kubectl get settings kubeconfig-default-token-ttl-minutes -oyaml
apiVersion: harvesterhci.io/v1beta1
default: "43200"
kind: Setting
metadata:
  annotations:
    harvesterhci.io/hash: d14a028c2a3a2bc9476102bb288234c415a2b01f828ea62ac5b3e42f
  creationTimestamp: "2024-06-04T10:22:40Z"
  generation: 3
  name: kubeconfig-default-token-ttl-minutes
  resourceVersion: "5902896"
  uid: 73b81673-7c41-4d4f-851e-359f5aeebe57
status:
  conditions:
  - lastUpdateTime: "2024-06-04T10:23:26Z"
    status: "False"
    type: configured
```

Reply:
the default setting of 0 is created on initialisation here: https://github.com/harvester/harvester/pull/5896/files#diff-f9454369fb0cd8a32b1a568916edfe94a77af272bf63bdc5dd63c976a200cf5dL38

Reply:
looks like setting got dropped while rebase of another PR 🤦 
i have added it back